lib/create-runner.js

```diff
@@ -131,18 +132,15 @@ export default function createRunner(options = {}, configFunc) {
 				if (typeof testPackages === "string") {
 					testPackages = testPackages.split(/\s+/);
 				}
-				// eslint-disable-next-line no-sync
-				fs.makeTreeSync(path.join(atomHome, "packages"));
+				await fs.makeTree(path.join(atomHome, "packages"));
 				for (const packName of testPackages) {
 					const userPack = path.join(userHome, "packages", packName);
 					const loadablePack = path.join(atomHome, "packages", packName);
 
 					try {
-						// eslint-disable-next-line no-sync
-						fs.symlinkSync(userPack, loadablePack, "dir");
+						await symlinkDir(userPack, loadablePack);
```

Comment:
It looks like they just use `junction` instead of `dir` for windows. If this works for you we could avoid importing a whole dependency.

```suggestion
						const type = process.platform === "win32" ? "junction" : "dir";
						await fs.symlink(userPack, loadablePack, type);
```

Reply:
The rest of that package is error handling when the folder already exists, but we don't have to worry about any of that since this will always be in a new temp folder.

Reply:
Duplicating the functionality of that package here means you need to add tests, verify that it works on all platforms, do error handling, etc. But using it as it is means it works without issues. This package is the package used inside pnpm. 
Not sure if a avoiding this a few KB of code helps the project

Reply:
I don't really see the need for new tests since the change would be so minimal. Literally this one line of code is all that is needed. Importing more dependencies means increasing the security risk. In this specific situation it doesn't seem necessary.

If we add the dependency we should add tests to make sure updating it doesn't break something in the future. This one line of code will most likely never need to be updated.

If we needed the error handling that would be one thing, but as I stated above we don't.

Reply:
I tried your suggestion, but many of the tests start to fail now.

Reply:
why are they failing? What error are they giving? I don't think it is because of that change because as I said that is all the dependency does.

Reply:
With symlink dir, it correctly creates links:

![image](https://user-images.githubusercontent.com/16418197/113542082-1d637100-95a9-11eb-9ab5-3cb2fdd6358d.png)


With your suggestion, it creates some empty files and the tests fail

![image](https://user-images.githubusercontent.com/16418197/113542091-218f8e80-95a9-11eb-844f-d76cc7b7f6bd.png)

The error:
```
TypeError [ERR_INVALID_CALLBACK] [ERR_INVALID_CALLBACK]: Callback must be a function. Received 'junction'
    at makeCallback (fs.js:150:11)
    at Proxy.symlink (fs.js:881:20)
    at file:///C:/Users/aminy/Documents/GitHub/JavaScript/atom-jasmine3-test-runner/lib/create-runner.js:142:16
    at Generator.next (<anonymous>)
    at file:///C:/Users/aminy/Documents/GitHub/JavaScript/atom-jasmine3-test-runner/lib/create-runner.js:290:22
    at Generator.next (<anonymous>)
    at step (file:///C:/Users/aminy/Documents/GitHub/JavaScript/atom-jasmine3-test-runner/lib/create-runner.js:83:37)
    at processTicksAndRejections (internal/process/task_queues.js:89:5) {
  rawStack: [
    CallSite {},
    CallSite {},
    CallSite {},
    CallSite {},
    CallSite {},
    CallSite {},
    CallSite {},
    CallSite {}
  ]
}
```

Reply:
I see. This is an issue with the way `fs-plus` makes `fs` async.

Reply:
try:

```js
// eslint-disable-next-line no-sync
const type = process.platform === "win32" ? "junction" : "dir";
fs.symlinkSync(userPack, loadablePack, type);
```

Reply:
The code will not speed up by using async functions since there will not be any other operations on the thread while it is waiting.